apiserver/root.go

```diff
@@ -119,6 +124,10 @@ type apiHandler struct {
 
 	// Deprecated: Resources are deprecated. Use WatcherRegistry instead.
 	resources *common.Resources
+
+	// controllerOnlyLogin is true if the client is using controller
+	// routes. Ultimately, this just indicates that no model UUID was specified
+	// in the request query (:modeluuid).
```

Comment:
Comment in the wrong place.

Reply:
I was going to pass it through more layers, this was incorrect. Thanks for picking this up.